### PR TITLE
8780 - Change email to not be null or empty

### PR DESCRIPTION
**Pull request by andrepestana-aot**
*Issue #:* /bcgov/entity#8780

*Description of changes:*

- Change email to not be null or empty

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/AddPeopleAndRoles/PeopleAndRoles.vue [hunk 1]
```diff
@@ -360,7 +362,7 @@ export default class PeopleAndRoles extends Mixins(EntityFilterMixin) {
       // remove the Completing Party role
       completingParty.roles = completingParty.roles.filter(role => role.roleType !== RoleTypes.COMPLETING_PARTY)
       // remove email address that we got from user profile
-      completingParty.officer.email = null
+      delete completingParty.officer.email // schema doesn't accept null or empty string
```
Comment (cameron-freshworks) [inline src/components/AddPeopleAndRoles/PeopleAndRoles.vue:365]: Interesting, haven't used this much.

**Review by cameron-freshworks — APPROVED** [review]
Looks good to me!

File: src/components/AddPeopleAndRoles/PeopleAndRoles.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/AddPeopleAndRoles/PeopleAndRoles.vue:365]: @andrepestana-aot Another way to do this is to remove this property when the IA is saved (if it's falsey). What do you think?

**Review by severinbeauvais — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "2.0.17",
+  "version": "2.0.18",
```
Comment (severinbeauvais) [inline package.json:3]: Another PR was just merged with this version number. You may have to update this. (I'm not sure why it's not showing as a conflict!?)

File: src/components/AddPeopleAndRoles/PeopleAndRoles.vue (hunk 1 above)
Comment (andrepestana-aot) [reply src/components/AddPeopleAndRoles/PeopleAndRoles.vue:365]: > @andrepestana-aot Another way to do this is to remove this property when the IA is saved (if it's falsey). What do you think?

Yes, either way work. I prefer this way.

Comment (severinbeauvais) [reply src/components/AddPeopleAndRoles/PeopleAndRoles.vue:365]: > Yes, either way work. I prefer this way.

Ok by me!

File: package.json (hunk 2 above)
Comment (andrepestana-aot) [reply package.json]: ok. Updated to 2.0.19 
anyway!

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**